cnap/redis/idxd-reset.patch

```diff
@@ -0,0 +1,26 @@
+From 07e1137a71021dc02df98b942a260b70ff852ca5 Mon Sep 17 00:00:00 2001
```

Comment:
All these are for redis images build, so maybe we cloud move all of them to ,container/cnap-redis/patches or something. Update the docker image then.

Reply:
ok.